src/main/java/uk/gov/justice/hmpps/datacompliance/config/DataComplianceProperties.java

```diff
@@ -21,4 +23,9 @@ public DataComplianceProperties(@Value("${elite2.api.base.url}") @URL final Stri
         this.elite2ApiBaseUrl = elite2ApiBaseUrl;
         this.elite2ApiOffenderIdsLimit = elite2ApiOffenderIdsLimit;
     }
+
+    @Bean
+    public TimeSource timeSource() {
+        return TimeSource.systemUtc();
```

Comment:
could you just use LocalDateTime and set timezone in Docker image e.g. `ENV TZ=Europe/London`?

Reply:
or are you having this so you can test e.g. `assertThat(offenderImageUpload.getValue().getUploadDateTime()).isEqualTo(DATE_TIME);` ?

Reply:
Yep it's so that it is testable - you have control over what you expect 'now' to be.